src/Symfony/Component/Form/Extension/Validator/EventListener/ValidationListener.php

```diff
@@ -26,6 +28,9 @@ class ValidationListener implements EventSubscriberInterface
     private ValidatorInterface $validator;
     private ViolationMapperInterface $violationMapper;
 
+    /** @var FormInterface[][] */
```

Comment:
```suggestion
    /** @var FormInterface[] */
```

Reply:
This is not a mistake. It's an array of arrays.

Similar to https://github.com/symfony/symfony/blob/def7ceb01acf32e872da4d74ddada2838ae7bf40/src/Symfony/Component/Form/AbstractExtension.php#L32